tests/console/mariadb_srv.pm

```diff
@@ -31,11 +31,18 @@ sub run {
     select_serial_terminal;
 
     zypper_call('in mariadb');
+    zypper_call "in policycoreutils-python-utils" if (is_jeos && has_selinux);
```

Comment:
Use `ensure_installed` and drop `is_jeos`? Maybe even unify the zypper calls.

Reply:
`ensure_installed` depends on x11 console, so can't be used here, and as for unifying zypper calls, the same test is ran also on mantenance jobs for SLE where selinux is not a thing

Reply:
> as for unifying zypper calls, the same test is ran also on mantenance jobs for SLE where selinux is not a thing

I mean something like

```
    my @pkgs = qw(mariadb);
    push @pkgs, "policycoreutils-python-utils" if has_selinux;
    zypper_call "in @pkgs";
```

No idea if there's any speed benefit

Reply:
only few seconds of the second zypper_call I guess?